ms2rescore/peptideshaker.py

```diff
@@ -152,21 +287,101 @@ def _parse_modification(modified_seq):
 
         return mods_peprec
 
+    @staticmethod
+    def _calculate_top7_peak_features(

```

Comment:
We should move this function to another module, to be shared with the maxquant pipeline, avoiding code duplication.

Reply:
Agreed, I didn't want to propose too large structural changes but yeah that is probably the way to go..